app/jobs/data_warehouse/stale_data_check_job.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module DataWarehouse
+  class StaleDataCheckJob < BaseJob
```

Comment:
I think `StaleDataCheckJob` is a bit of a misleading name, this is exporting table info that can later be used for a stale data check, but it's not going to check the stale data itself.

WDYT of `TableSummaryExportJob`? or something along those lines
```suggestion
  class TableSummaryExportJob < BaseJob
```